src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -8,74 +8,93 @@
+    }
+
     private Ladder(List<HorizontalLine> horizontalLines) {
         this.ladder = new ArrayList<>(horizontalLines);
     }
 
-    private void assignVertical(LineSelector lineSelector) {
-        int width = ladder.get(0).size();
+    private void assignOneLineToEachVerticalLines(int width,
+                                                  LineSelector lineSelector) {
         int beforeSelectedIndex = -1;
         for (int i = 0; i < width; i++) {
-            beforeSelectedIndex = assignVertical(i, beforeSelectedIndex,
-                    lineSelector);
+            beforeSelectedIndex =
+                    assignOneLineToVerticalLine(i, beforeSelectedIndex,
+                            lineSelector);
         }
     }
 
-    private int assignVertical(int index,
-                               int beforeSelectedIndex,
-                               LineSelector lineSelector) {
+    private int assignOneLineToVerticalLine(int index,
+                                            int beforeSelectedIndex,
+                                            LineSelector lineSelector) {
         int height = ladder.size();
         int selectedIndex = lineSelector.select(height);
         if (beforeSelectedIndex == selectedIndex) {
             selectedIndex = (selectedIndex + 1) % height;
         }
         HorizontalLine horizontalLine = ladder.get(selectedIndex);
-        horizontalLine.makePointToTrue(index);
+        horizontalLine.makeDirection(index);
         return selectedIndex;
     }
 
-    private void assignHorizontal(RightDirection rightDirection) {
+    private void assignLinesToEachHorizontalLines(
+            RightDirection rightDirection) {
         for (HorizontalLine horizontalLine : ladder) {
-            assignHorizontal(horizontalLine, rightDirection);
+            assignLinesToHorizontalLine(horizontalLine, rightDirection);
         }
     }
 
-    private void assignHorizontal(HorizontalLine horizontalLine,
-                                  RightDirection rightDirection) {
+    private void assignLinesToHorizontalLine(HorizontalLine horizontalLine,
+                                             RightDirection rightDirection) {
         for (int i = 0; i < horizontalLine.size(); i++) {
             boolean isAbleToRight = rightDirection.isAbleToRight();
-            horizontalLine.makePointTo(i, isAbleToRight);
+            horizontalLine.makeDirectionTo(i, isAbleToRight);
         }
     }
 
-    public int height() {
-        return ladder.size();
-    }
-
-    public List<Point> vertical(int index) {
+    public List<Point> getVerticalLine(int index) {
         return ladder.stream()
                 .map(horizontalLine -> horizontalLine.getPoint(index))
                 .collect(Collectors.toList());
     }
 
-    public HorizontalLine horizontal(int index) {
+    public HorizontalLine getHorizontalLine(int index) {
         return ladder.get(index);
     }
 
-    public int size() {
+    public int height() {
```

Comment:
```suggestion
    public int getHeight() {
```
height 라는 이름은 정확한 행동을 예측하기 어려울 것 같아요.

Reply:
`size`는 지금 보니 동사여서 사용이 가능했던 것이였군요 😃